damus/en-US.lproj/Localizable.stringsdict

```diff
@@ -13,9 +13,15 @@
 			<key>NSStringFormatValueTypeKey</key>
 			<string>d</string>
 			<key>zero</key>
-			<string>0 other notes</string>
+			<string>%d other notes</string>
 			<key>one</key>
-			<string>1 other note</string>
+			<string>%d other note</string>
+			<key>two</key>
```

Comment:
These are not issues for English, but other languages have different pluralization rules when the count is `two`, `few`, or `many`.

Reply:
Never mind. These get stripped out on export for English. I think Transifex knows how to add the rules for other languages. I'll revert this.